src/capital-project/capital-project.service.ts

```diff
@@ -30,21 +30,48 @@ export class CapitalProjectService {
     private readonly agencyBudgetRepository: AgencyBudgetRepository,
   ) {}
 
+  checkCommitmentsTotalRangeIsValid(
+    commitmentsTotalMin: number,
+    commitmentsTotalMax: number,
+  ): boolean | undefined {
+    if (commitmentsTotalMin > commitmentsTotalMax) return undefined;
+    return true;
+  }
+
   async findMany({
     limit = 20,
     offset = 0,
     cityCouncilDistrictId = null,
     communityDistrictCombinedId = null,
     managingAgency = null,
     agencyBudget = null,
+    commitmentsTotalMin = null,
+    commitmentsTotalMax = null,
   }: {
     limit?: number;
     offset?: number;
     cityCouncilDistrictId?: string | null;
     communityDistrictCombinedId?: string | null;
     managingAgency?: string | null;
     agencyBudget?: string | null;
+    commitmentsTotalMin?: string | null;
+    commitmentsTotalMax?: string | null;
   }) {
+    const min = commitmentsTotalMin
+      ? parseFloat(commitmentsTotalMin.replaceAll(",", ""))
+      : null;
+    const max = commitmentsTotalMax
+      ? parseFloat(commitmentsTotalMax.replaceAll(",", ""))
+      : null;
+
+    if (
+      min !== null &&
+      max !== null &&
+      this.checkCommitmentsTotalRangeIsValid(min, max) === undefined
```

Comment:
This is just preference. But, I think it's a little cleaner to remove the `checkCommitmentsTotalRangeIsValid` and easier to read with the logical check in-line.

```suggestion
      min > max
```